meetings/2017-11-05.md

```diff
@@ -0,0 +1,117 @@
+# Node.js Foundation Community Committee Meeting 2017-05-11
+
+## Links
+
+* **GitHub Issue**: [TSC#227](https://github.com/nodejs/TSC/issues/227)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1urhinDNJK-EEMziNob8-mlsOEZM_zecluDIgBA6kVOw/
+* _Previous Minutes Google Doc_: https://docs.google.com/document/d/1YevHuA_csoTltDJ6mHr21fetZshNqWpsRbBRkSHhmYE/
+## Present
+- Ashley Williams ([ashleygwilliams](https://github.com/ashleygwilliams))
+- Bryan Hughes ([nebrius](https://github.com/nebrius))
+- Gregor Martynus ([gr2m](https://github.com/gr2m))
+- Tracy Hinds ([hackygolucky](https://github.com/hackygolucky))
+- William Kapke ([williamkapke](https://github.com/williamkapke))
+- Tierney Coren ([bnb](https://github.com/bnb))
+- Michael Dawson 
+- Penny 

```

Comment:
In addition to @mhdawson's comment, add:
`Penny ([pup](https://github.com/pup))`